### ISSUE TITLE
pathlib.PosixPath is completely broken

### ISSUE THREAD

**Issue by KOLANICH** [graalvm/graalpython#48]
```python
from pathlib import Path
p = Path("/usr")
str(p) # __str__ returned non-string (type NoneType)
p.absolute() # None
p / "local" # None
```

**Comment by fangerer** [comment]
Hi Kolanich,
thank you for reporting this issue.
I can reproduce the problem and a fix is on the way. Unfortunately, I cannot run the full pathlib tests from CPython due to other problems. Therefore, I cannot guarantee that everything will work as expected.

**Comment by fangerer** [comment]
The fix is merged in the current master (revision: 82ff5e7f2451fb9c35b9ce7c81e13cd35e8764a0) and should go into the next GraalVM release which is expected for Feb 1.

**Comment by timfel** [comment]
The fix is included in rc12.
